source/introduction/introduction.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+## What is the technical documentation tool?
+
+This tool is a template for technical and developer documentation. 
+
+It uses the [Middleman static site generator](https://middlemanapp.com/) [external link] to convert markdown files into HTML-based documentation that looks consistent with the GOV.UK and GDS style and format. 
+
+It is suitable for multiple styles of documentation, including API documentation. 
+
+## Why should you use this?
+
+It imposes consistency on technical documentation which should reassure end users.
```

Comment:
Only minor comment here: I'm not sure about using the word 'impose'

Reply:
how about promote?

Reply:
> how about promote?

 💯